Assets/MirageXR/Player/Scripts/Mobile/ContentEditors/GlyphEditorView.cs

```diff
@@ -21,11 +21,16 @@ public class GlyphEditorView : PopupEditorBase
     [SerializeField] private Button _btnNextStep;
     [SerializeField] private Button _btnPreviousStep;
     [SerializeField] private ActionObject[] _actionObjects;
+    [Space]
+    [SerializeField] private Button _accept;
+    [SerializeField] private GameObject _acceptBtnOb;
+    [SerializeField] private GameObject _closeBtnOb;
```

Comment:
pleace use full name
```suggestion
    [SerializeField] private GameObject _acceptButtonObject; 
    [SerializeField] private GameObject _closeButtonObject;
```